.github/workflows/nvm.yml

```diff
@@ -5,9 +5,47 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: node --version
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - run: |
           nvm install --latest-npm --no-progress
-          echo "::set-env name=PATH::${NVM_BIN}:${PATH}"
+          echo "$(dirname $(which node))" >> $GITHUB_PATH
```

Comment:
[suggest] Let's use quotes for environment variables.

```suggestion
          echo "$(dirname $(which node))" >> "$GITHUB_PATH"
```

Reply:
Yeah normally I would do this but in the docs for GitHub's "magic" environment variables, they don't include quotes